### PR TITLE
#163225414 ft-signup-email-confirmation-redirect

### PR DESCRIPTION
**Pull request by SokoPaulSokool**
### What does this PR do?
This pull request enables one to be redirected to the frontend when one clicks the on the signup verification link.

### Description of tasks to be completed?
Add frontend domain name to the environment variables with key `FRONT_END_DOMAIN`
Write code to redirect to `/login` if the link is right
Write code to redirect to `/invalid_link` if the link is invalid

## How should this be tested?
Use https://ah-backend-athena-stagin-pr-39.herokuapp.com/ to create a user and validate him using the link sent it the email.

The link should redirect you to the login page

### Screen shots
screen shot 2019-01-16 at 19 26 10

screen shot 2019-01-16 at 19 56 09

### CODE REVIEW

**Review by kafuuma — APPROVED** [review]

**Review by bibangamba — COMMENTED** [review]
Quick question on this, I see the screenshot says `404 not found` but shouldn't you handle the verification process on the frontend as well? I dunno, just asking an open question. maybe @JaymesKat can help with an opinion. If you've already implemented what I am talking about though, just help us add a screenshot of what the frontend verification page would look like.
Just wondering if the handling of the frontend URL should happen in this PR too or a new one (same thing for password reset callback URL change)

Otherwise I am approving because the task mentioned in the story is complete.

**Review by bibangamba — APPROVED** [review]

**Comment by Kasulejoseph** [comment]
type in PR `/lohin`

**Comment by SokoPaulSokool** [comment]
I have edited it

**Review by Kasulejoseph — APPROVED** [review]

File: app.json [hunk 1]
```diff
@@ -51,20 +51,23 @@
...
       "quantity": 1
     }
   },
-  "addons": [
-    "heroku-postgresql"
-  ],
+  "addons": ["heroku-postgresql"],
   "scripts": {
-    "postdeploy": "./manage makemigrations&&./manage.py migrate --noinput"
+    "postdeploy": "./manage.py makemigrations&&./manage.py migrate --noinput"
```
Comment (JaymesKat) [inline app.json]: This change is outside the scope of what you are working on. @SokoPaulSokool Wasn't this fixed in another PR?

Comment (SokoPaulSokool) [reply app.json]: I have fixed it

**Review by JaymesKat — APPROVED** [review]
